cryptofeed/exchanges/binance.py

```diff
@@ -500,7 +500,7 @@ async def _order_update(self, msg: dict, timestamp: float):
             BUY if msg['S'].lower() == 'buy' else SELL,
             msg['X'],
             LIMIT if msg['o'].lower() == 'limit' else MARKET if msg['o'].lower() == 'market' else None,
-            Decimal(msg['Z']) / Decimal(msg['z']) if not Decimal.is_zero(Decimal(msg['z'])) else None,
```

Comment:
`Decimal(msg['Z']) / Decimal(msg['z']) if not Decimal.is_zero(Decimal(msg['z'])) else Decimal(msg['p']),`

Reply:
As we are only placing limit orders do we really need to check this?.